tests/src/actionContainers/NodeJsActionContainerTests.scala

```diff
@@ -306,4 +306,50 @@ class NodeJsActionContainerTests extends FlatSpec with Matchers {
         filtered(out).trim shouldBe empty
         filtered(err).trim shouldBe empty
     }
+
+    it should "error when requiring a non-existent package" in {
+        val (out, err) = withNodeJsContainer { c =>
+            val code = """
+                | function main(args) {
+                |     require('wildlyuniquenameofanonexistentpackage');
```

Comment:
I really like the idea to have such a test - but I think this test would fail if I register a package called "wildlyuniquenameofanonexistingpackage" in npm which everyone could do. Maybe could require something with an illegal name instead ? - The requirements for a valid npm module name are described here https://docs.npmjs.com/files/package.json